### PR TITLE
FIX cache not cleared on node visibility change

### PR DESCRIPTION
**Pull request by ITernovtsiy**
Fix issue with cache not cleared on node visibility change.
Search index takes wrong visibility status from cache.

Todo:
- [x] 2017.12
- [x] JIRA issue - https://jira.ez.no/browse/EZP-30493

### CODE REVIEW

File: kernel/content/ezcontentoperationcollection.php [hunk 1]
```diff
@@ -1017,6 +1017,12 @@ static public function changeHideStatus( $nodeID )
                 eZContentObjectTreeNode::hideSubTree( $curNode );
         }
 
+        // notify cache system about visibility change
+        ezpEvent::getInstance()->notify('content/cache', array(
```
Comment (andrerom) [inline kernel/content/ezcontentoperationcollection.php]: maybe move this inside ` if ( is_object( $curNode ) )`? otherwise nothing is actually changed

**Comment by andrerom** [comment]
Updated PR here @diaborn19, would you be able to move the changes either with the two commits or just one by you to 2017.12 branch?

**Comment by ITernovtsiy** [comment]
Hi @andrerom 
Sorry, missed your comment, changed to 2018.09

**Comment by andrerom** [comment]
I meant 2017.12 here, it's the release used with eZ Platform 1.13.x, so the oldest branch here that receives fixes for the time being. But if you prefer I can adapt this on merge.

**Comment by ITernovtsiy** [comment]
Hi, EZP-30493 created and base branch switched to 2017.12. Thanks.

**Pull request merged**